impeller/tools/malioc.json

```diff
@@ -13183,7 +13181,7 @@
       "type": "Compute",
       "variants": {
         "Main": {
-          "fp16_arithmetic": 100,
+          "fp16_arithmetic": 0,
```

Comment:
I guess the compute shaders need this specification as well?

Reply:
all of the compute shaders are dealing with positional data so defaulting to mediump is probably a bad idea